### PR TITLE
Docker testing env docs

### PR DESCRIPTION
**Pull request by elpaso**
The file name and position is provisional because I don't have yet a clear idea about where it will eventually land. 

But the important thing is that the documentation exists somewhere and is (hopefully) accurate, so, here it is.

### CODE REVIEW

**Comment by elpaso** [comment]
@Gustry here you go

File: .docker/README.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+to simulate the installation of the plugin inside QGIS, you'll need to
+make an additional step: call `qgis_setup.sh <YourPluginName>` in the
+docker container before actually running the tests (see the paragraph
+about Running on Travis for a complete example).
+
+The `qgis_setup.sh` script prepares QGIS to run in headless mode and
+simulate the plugin installation process:
+
+- creates the QGIS profile folders
+- "installs" the plugin by making a symbolic link from the prfiles folder to the plugin folder
```
Comment (Gustry) [inline .docker/README.md]: `profiles`

**Review by Gustry — COMMENTED** [review]
Nice ! Thanks.
Should we provide a example of the `run_all` function? with the test discovery.

I gave a quick look only, I will try later on the inasafe plugin

File: .docker/README.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+└── test_TravisTest.py
+```
+
+To run the tests inside the container, you must mount the directory that
+contains the tests (e.g. your local directory `/my_tests`) into a volume
+that is accessible by the container, see `-v /my_tests/:/tests_directory`
+in the example below:
+
+```
+$ docker run -d --name qgis -v /tmp/.X11-unix:/tmp/.X11-unix \
```
[the remark this answers is not in the document]
Comment (elpaso) [reply .docker/README.md]: no, I don't want to write a unix admin manual...

**Pull request merged**